# Lewis Quayle, PhD

<img align='right' src="https://github.com/lquayle88/lquayle88/blob/main/lquayle.png?raw=true" width="325">

## Cancer Bioinformatician and Data Scientist

I have over nine years of experience working in the academic research setting analysing quantitative data from a wide variety of technologies, including next-generation sequencing, microarray, NanoString nCounter® and high-throughput RT-qPCR array profiling platforms, amongst others.

I am skilled in development and deployment of scalable, automated cloud-based or high-performance computing (HPC) workflows, as well as being proficient in acquiring, cleaning and wrangling data from a broad range of sources, computing descriptive metrics and undertaking exploratory data analyses, implementing inferential statistical methods, and utilising best practice graphical modelling and data visualisation techniques. Check out my <a href="https://blog.lewisdoesdata.com/portfolio/" target="_blank">portfolio</a>, my <a href="https://blog.lewisdoesdata.com" target="_blank">blog</a> or my <a href="https://www.linkedin.com/in/lewis-quayle" target="_blank">LinkedIn</a> profile to see some of the things I have done in the past.

My current research activity is centered on a project aiming to create a mutational map of the non-coding regions of patient genomes for the bladder cancer arm of the <a href="https://www.genomicsengland.co.uk/initiatives/100000-genomes-project" target="_blank">100,000 Genomes Project</a>. I am also contributing bioinformatics capability to the <a href="https://fundingawards.nihr.ac.uk/award/NIHR128103" target="_blank">Genotype of Urothelial Cancer - Stratified Treatment and Oncological outcomes (GUSTO) phase II clinical study</a>. These roles involve assessing, testing, and implementing best practice methods for genomic data analysis, as well as contributing to the maintenance of well-curated, highly structured and transparent resources for integrating genomic and clinical data.

I am currently a member of the <a href="https://sbc.shef.ac.uk./" target="_blank">Sheffield Bioinformatics Core Facility</a>. In this role, I am expected to deliver custom analyses, processed data, experimental results, and reports detailing quality control procedures and workflows carried out in secure containerised environments to core facility users. I am also involved in supporting the provision of best-practice training and workshops in next-generation sequencing and bioinformatics techniques to laboratory scientists, assisting investigators across the university in statistically sound experimental design, aiding interpretation of results, and grant or manuscript writing where these require extensive bioinformatics input.

### Skills Overview:

- &#x1F4BB; &nbsp; R • Unix Shell (BASh) • Python • SQL

- &#x1F310; &nbsp; HTML • CSS • Jekyll

- &#x1F6E2; &nbsp; SQL • Excel • PowerBI

- &#x1F5A5; &nbsp; R Shiny • Shiny Dashboard • Plotly • GraphPad Prism

- &#x1F6E0; &nbsp; Git • Markdown • Nextflow • Conda • Docker • Singularity

<br>
<h2 align="center">Contact Me</h2>
<p align="center">
  <a  href="https://www.linkedin.com/in/lewis-quayle" target="_blank" >
    <img src="https://www.vectorlogo.zone/logos/linkedin/linkedin-icon.svg" alt="LinkedIn Profile" height="30" width="30">
  </a>
  &nbsp;
  <a  href="mailto:drlquayle@gmail.com" target="_blank" >
    <img src="https://www.vectorlogo.zone/logos/gmail/gmail-icon.svg" alt="Gmail Address" height="30" width="30">
  </a>
  
  
